src/main/java/com/example/UserService.java

```diff
@@ -0,0 +1,29 @@
+package com.example;
+
+import java.util.List;
+
+public class userService {
```

Comment:
Class name starting with lowercase...i think UserService would be more appropriate